app/src/main/kotlin/com/antosik/benchproject/app/popular/movies/viewModel/PopularMoviesViewModel.kt

```diff
@@ -39,6 +41,13 @@ class PopularMoviesViewModel @Inject constructor(
         loadPopularMovies()
     }
 
+    fun updatePopularMovie(movie: Movie) {
+        viewModelScope.launch {
+            updatePopularMovieUseCase(movie.toDomain())
```

Comment:
@BorysChajdas @Nataniel-Antosik  what do You think about doing this two operations in a use case :) ? As we probably want to avoid chaining operations in viewmodel

Reply:
You consider the situation when in `updatePopularMovieUseCase` I should call another use case for this example `getPopularMoviesUseCase`?

Reply:
To be honest I didn't see how it should work because get use case to return some value and it is handled in ViewModel now I invoked the method in a very simple way to refresh data, I believe somehow I can do this better.

Reply:
Maybe I have an idea of how to deal with it now I can create an issue to create a better way to handle updating movies in the database. Probably I discussed it with @BorysChajdas some days ago 😄

Reply:
The main goal here is to have a very simple logic in VM, and handle all cases in use case. This is quite simple case but please imagine that You will have 3 or 4 operations, one after another. We can get back to this, when You will start working on this issue as a separate task :D

Reply:
I created an issue for this #177